readit/cli.py

```diff
@@ -129,28 +129,28 @@ def option_yes_no(url):
     """
     Asks whether to bookmark invalid URLs or Offline URLs to database.
     """
-    option = input("Still you want to bookmark: Yes/No --> ")
-    if option == "Yes" or option == "Y" or option == "y":
+    option = input("Still you want to bookmark: Yes/No ? ")
+    if option == "Yes" or option == "Y" or option == "y" or option == "yes":
```

Comment:
@shitalmule04 actually I'm agree with @ganeshhubale. You don't need to use `elif`.
But you can try this solution for that.
```
option = input("Still you want to bookmark (yes/no):")
if option.lower() in ['yes', 'y']:
   database_connection.add_url(url)
```

Reply:
@amolkahat, Thanks.